includes/Extension/SearchReplace/class-replace.php

```diff
@@ -85,12 +96,163 @@ public function add_replace_button() {
+				// The plugin is already active.
+				// $instawp_connect = menu_page_url( 'instawp-connect', false );				
+				$instawp_connect = menu_page_url( 'instawp', false );				
+				$button = array(
+					'message' => esc_attr__( 'Create a Staging Site', 'string-locator' ),
+					'url'     => $instawp_connect,
+					'classes' => array( 'string-locator-instawp-button', 'disabled' ),
+				);
+
+				if ( '' !== $activated ) {
+					$button['message'] = esc_attr( $activated );
+
+					$button['target'] = 'target="_blank"';
+				}
+				$button['logo-img'] = '<span class="btn-logo-inline"><img src="'. plugins_url( 'views/assets/instawp-logo.svg', STRING_LOCATOR_PLUGIN_FILE ).'" alt="InstaWP logo"></span>';
+
+			} elseif ( self::is_plugin_installed( $plugin_slug ) ) {
+				$url = self::is_plugin_installed( $plugin_slug );
+
+				// The plugin exists but isn't activated yet.
+				$button = array(
+					'message' => esc_attr__( 'Create a Staging Site', 'string-locator' ),
+					// 'url'     => $url,
+					'url'     => "javascript:void 0;",
+					'classes' => array( 'instawp-activate-now' ),
+				);
+
+				$button['logo-img'] = '<span class="btn-logo-inline"><img src="'. plugins_url( 'views/assets/instawp-logo.svg', STRING_LOCATOR_PLUGIN_FILE ).'" alt="InstaWP logo"></span>';
+				if ( '' !== $activate ) {
+					$button['message'] = esc_attr( $activate );
+				}
+			} else {
+				// The plugin doesn't exist.
+				$url    = wp_nonce_url(
+					add_query_arg(
+						array(
+							'action' => 'install-plugin',
+							'plugin' => $plugin_slug,
+						),
+						self_admin_url( 'update.php' )
+					),
+					'install-plugin_' . $plugin_slug
+				);
+				$button = array(
+					'message' => esc_attr__( 'Create a Staging Site', 'string-locator' ),
+					'url'     => $url,
+					'classes' => array( 'sl-instawp-install-now', 'install-now', 'install-' . $plugin_slug ),
+				);
+
+				if ( '' !== $install ) {
+					$button['message'] = esc_attr( $install );
+				}
+
+				$button['logo-img'] = '<span class="btn-logo-inline"><img src="'. plugins_url( 'views/assets/instawp-logo.svg', STRING_LOCATOR_PLUGIN_FILE ).'" alt="InstaWP logo"></span>';
+			}
+
+			if ( ! empty( $classes ) ) {
+				$button['classes'] = array_merge( $button['classes'], $classes );
+			}
+
+			$button['classes'] = implode( ' ', $button['classes'] );
+
+			?>
+			<span class="plugin-card-<?php echo esc_attr( $plugin_slug ); ?>" style="float: right; margin-top: 7px;">
+				<?php echo !empty($button["logo-img"]) ? $button["logo-img"] : '';?>
+				<a href="<?php echo esc_url( $button['url'] ); ?>" class="<?php echo esc_attr( $button['classes'] ); ?>" data-originaltext="<?php echo esc_attr( $button['message'] ); ?>" data-name="<?php echo esc_attr( $plugin_name ); ?>" data-slug="<?php echo esc_attr( $plugin_slug ); ?>" aria-label="<?php echo esc_attr( $button['message'] ); ?>" <?php echo !empty($button["target"]) ? $button["target"] : '';?>> 
```

Comment:
The use of links for interactive buttons is not good for accessibility. These should be replaced with appropriate buttons and button styling instead. (this references the JavaScript later down the PR which grabs the links and treats them as buttons).

Reply:
Resolved In latest Commit.